lib/pure/collections/sequtils.nim

```diff
@@ -43,6 +43,20 @@ proc concat*[T](seqs: varargs[seq[T]]): seq[T] =
       result[i] = itm
       inc(i)
 
+proc count*[T](s: seq[T], x: T): int =
```

Comment:
This is a nit, but let's use more descriptive param names: ``list`` and ``item`` perhaps?

Reply:
I've used `s` and `x` names, like some other procedures have (e.g. `cycle` and `repeat`). I can change it to proposed `list` and `item`, but that would stand out. Some other procedures use `data`, so maybe this might be used instead of `list`.

Thoughts?

Reply:
Yeah, I realise it's out of the ordinary. Other procedures should use more descriptive names as well. I think ``list`` makes more sense than ``data``.

By the way, another thought: you might wish to consider replacing ``seq`` with ``openarray`` so that arrays and sequences are accepted by your procedure.

Reply:
> you might wish to consider replacing seq with openarray so that arrays and sequences are accepted by your procedure.

Will do.

> I think list makes more sense than data.

Ok, I'll change it to `list` and `item`.

> Other procedures should use more descriptive names as well. 

I can change this too while I'm at it.